### DISCUSSION TITLE
【意见征求】如何翻译focal distance？

### DISCUSSION THREAD

**Discussion by kanition** [kanition/pbrtbook#70] [问答]
请教大家如何翻译focal distance？
原文在[这里](https://www.pbr-book.org/3ed-2018/Camera_Models/Projective_Camera_Models#eq:thin-lens-focus)

注意前文还有一个表示为f的focal length，已经译为“焦距”。
本想将focal distance翻译为“对焦距离”，但似乎“对焦距离”的含义（见[百度百科](https://baike.baidu.com/item/%E5%AF%B9%E7%84%A6%E8%B7%9D%E7%A6%BB/1104718)，定义为物象距离 ）与本书的定义（像距）不是一回事，所以更加不确定了。

欢迎各位读者给予帮助！

**Comment by a690089735** [comment]
https://www.cambridgeincolour.com/forums/thread2456.htm 我找到了这个,但是我看不懂.
推测像是物距和相距的区别?
不过gg搜索focal distance,出来的结果仍然是 Focal length
(虽然只会20个英文字母,却还想掺和一下,就当我是来活跃气氛的吧😀)

**Reply by kanition** [reply]
谢谢，这个我也搜到过，不过似乎也说的不清楚。可能本书原文也不是用得很严谨，继续观望吧

**Comment by kanition** [comment]
> We call the distance between the projection point and the plane where everything is in perfect focus the focus distance. Be aware that the focus distance is not the same as the focal length — the focal length is the distance between the projection point and the image plane.

[_Ray Tracing in One Weekend_](https://raytracing.github.io/books/RayTracingInOneWeekend.html#defocusblur)

即该材料中focal distance则表示物距，但专门强调了与focal length的差别，有更高可信度。

**Reply by tianlei124** [reply]
他说的看起来不是 focal distance 而是 focus distance
